src/server/.tests/routes/controllers/Middlewares/logInUser.spec.js

```diff
@@ -0,0 +1,54 @@
+const logInUser = require('../../../../routes/controllers/Middlewares/logInUser')
+const assert = require('chai').assert
+const sinon = require('sinon')
+
+let suite = {}
+
+describe('logInUser', () => {
+	beforeEach(() => {
+		suite = {}
+		suite.passportMock = createPassportMock()
+		suite.loginUserMock = logInUser(suite.passportMock)
+		suite.responseMock = sinon.stub()
+		suite.nextSpy = sinon.spy()
+		suite.authenticatio
+
+	})
+	it('should call next() without calling authenticationMiddleware, when req.isAuthenticated is true', () => {
+		//given
+		const requestMock = {
+			isAuthenticated: () => true
+		}
+		
+		//when
+		suite.loginUserMock(requestMock, suite.responseMock, suite.nextSpy)
+
+		//then
+		const authenticateSpy = suite.passportMock.authenticate
+		assert.isTrue(suite.nextSpy.calledOnce)
+		assert.isFalse(authenticateSpy.calledOnce)
+	})
+
+	it('should call passport.authentiate middleware, when req.isAuthenticated is false ', () => {
```

Comment:
Give it a description from business POV, as the reason for this behavior is unclear

Reply:
ok